docs/proposals/mvp-dashboard-design.md

```diff
@@ -10,16 +10,37 @@ Last updated: 2017-01-12
 
 ## Abstract
 
-A short summary of the proposal.
+We propose a dashboard/server that allows for the automated retrieval,
+archive, format, and display of performance tests run in the performance lab.
+This tool will allow our team to investigate performance issues and facilitate
+the sharing of lab results with other teams.
```

Comment:
@meteorfox: Thanks for the comments.  Here's my feedback.
1. Made the change concerning the actions the system does: retrieve and display 
1. I mean "format" as, the manner of displaying the data is changed from a json return, to a graph.
1. runs -> results
1. Okay, I'll change the wording so that it is understood that this tool is for all labs. I will change the wording about 'investigation' to identifying regressions. So this would be a monitoring system.